tools/prepare.js

```diff
@@ -73,13 +72,20 @@ async function prepareInTemp({ majorChrome, workPath, headRevision, definitionsR
   // default. This means running newer versions of the tool on older IDL files
   // will generate the wrong output. We therefore instead use a known-good
   // version of the tool on any IDL files from before Chrome 121.
-  const toolRevision = typeof majorChrome === "number" && majorChrome < 121
+  const useOldToolRevision = typeof majorChrome === "number" && majorChrome < 121;
+
+  const toolRevision = useOldToolRevision
```

Comment:
Why create an alias to the value?

Reply:
This isn't an alias, on the next lines there is a ternary to assign to it :)